auto_test:
  ros__parameters:
    test_cases:
      - MakeGoalDiagonalLeft
      - MakeGoalDiagonalRight
    repeats: 1
    loop: true
